app/forms/ReplyToMessage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import play.api.data.Form
+import play.api.data.Forms.{mapping, nonEmptyText}
+
+case class ReplyToMessage(messageReply: String)
+
+object ReplyToMessage {
+
+  val form: Form[ReplyToMessage] = Form(mapping("messageReply" -> nonEmptyText)(ReplyToMessage.apply)(ReplyToMessage.unapply))
```

Comment:
Not sure we should be doing any validation like this 'nonEmptyText' validation. DC will do this at their end and return to us a partial with the form error per-rendered.

Reply:
This is just temporary. Let's wait to actually know what DC expect in the payload of the 'POST Reply'.

Reply:
Just got off a call with DC and they confirmed that they will do the validation their end and return the conversation partial with any form errors rendered as part of the POST's response. So we definitely will not need this validation our end.

Reply:
Ok, removed.